### PR TITLE
Run integration-tests in GH workflows

### PR DESCRIPTION
**Pull request by MartinBasti**
Requires: https://github.com/release-engineering/cachito/pull/512

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -44,7 +44,7 @@ run run-start:
 	# - nexus needs to write to ./tmp/nexus-data
 	# - integration tests need to write to (and create) ./tmp/cachito-archives
 	chmod -R 0777 ./tmp
-	$(CACHITO_COMPOSE_ENGINE) up
+	$(CACHITO_COMPOSE_ENGINE) up -d
```
[the remark this answers is not in the document]
Comment (MartinBasti) [reply Makefile]: updated

**Review by akhmanova — APPROVED** [review]
LGTM! Thank you :)

File: tests/integration/conftest.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 import pytest
 import yaml
 
-import utils
+from . import utils
```
Comment (brunoapimentel) [inline tests/integration/conftest.py:9]: Just curious, why is importing like this better?

File: .github/workflows/gating.yaml [hunk 3]
```diff
@@ -105,3 +105,29 @@ jobs:
...
         ignore: DL3041
         failure-threshold: warning
+
+  integration_tests:
+    runs-on: ubuntu-latest
+    needs:
+    - tests
+    - flake8
+    - black
+    - hadolint
```
Comment (brunoapimentel) [inline .github/workflows/gating.yaml:115]: It seems we need to fix our Dockerfiles now :smile:

**Review by brunoapimentel — APPROVED** [review]

**Comment by brunoapimentel** [comment]
LGTM :100: 

I'm just wondering if we can run the `cached_dependencies` tests on Github. The default test configuration just skips them.

The only problem I found running them locally is that, since my timezone is UTC-3, Athens gets some inconsistencies comparing the timestamps from the deps it downloads, or something of the like. Setting it to UTC makes all `cached_dependencies` test run fine. This should be a no brainer in the Github env.

Anyway, just putting the idea out there, I don't think this is something to be done in this PR,

**Comment by akhmanova** [comment]
@brunoapimentel 
> I'm just wondering if we can run the cached_dependencies tests on Github. The default test configuration just skips them.
Hm, I think there is another problem: we need to grant writing access to (cachito-testing org)[https://github.com/cachito-testing]

File: tests/integration/conftest.py (hunk 2 above)
Comment (MartinBasti) [reply tests/integration/conftest.py:9]: It didn't work for me without it. python was not able to find module without installing cachito. Why cachito should be installed for int tests?

**Pull request merged**